packages/safe-core-sdk-types/src/ethereumLibs/EthAdapter.ts

```diff
@@ -19,8 +19,8 @@ export interface EthAdapterTransaction {
   from: string
   data: string
   value?: string
-  gasPrice?: number
-  gasLimit?: number
+  gasPrice?: number | string
+  gasLimit?: number | string
```

Comment:
Why is `number` allowed?
I guess same happens with `maxFeePerGas` and `maxPriorityFeePerGas` in lines 24-25

Reply:
As this type is used always as input typing for the providers I left it a bit more flexible. Both libraries would admit
```undefined | number | string | BN/BigNumber```

We are already removing the BN/BigNumber possibility.